modules/core/src/shaderlib/project/viewport-uniforms.js

```diff
@@ -107,27 +111,27 @@ function calculateMatrixAndOffset({
     case PROJECT_COORDINATE_SYSTEM.LNGLAT_OFFSETS:
     case PROJECT_COORDINATE_SYSTEM.METER_OFFSETS:
     case PROJECT_COORDINATE_SYSTEM.LNGLAT_AUTO_OFFSET:
+    case PROJECT_COORDINATE_SYSTEM.IDENTITY:
       // Calculate transformed projectionCenter (using 64 bit precision JS)
       // This is the key to offset mode precision
       // (avoids doing this addition in 32 bit precision in GLSL)
-      const positionPixels = viewport.projectFlat(
+      const positionCommonSpace = viewport.projectPosition(
         shaderCoordinateOrigin,
         Math.pow(2, coordinateZoom)
       );
 
       cameraPos = [
```

Comment:
Worth being consistent if we are making these changes? 

I.e. `cameraPosCommon`?